projects/common/src/constants/application-constants.ts

```diff
@@ -2,7 +2,7 @@ import { InjectionToken } from '@angular/core';
 
 export const GLOBAL_HEADER_HEIGHT = new InjectionToken<string>('Global Header Height');
 
-export const enum ApplicationFeature {
```

Comment:
Why did we remove const here

Reply:
Because it was causing issues with the FF changing being registered when I was testing it, however after more reading and testing it was because of the compute vs. non-compute nature of enums and how i was testing it. 

I was just refreshing the tab, but since non-comput enums(const) are read at compile time, it requires shutting down the app entirely and rebuilding for the FF changes to take effect.  I'll change this back as it should be.

Reply:
Hmm - const enums are defined compile time, but changing them results in a new compilation, just like any other change. So they should behave in the same way assuming you're using a basic `ng serve` or equivalent. May want to investigate your local setup at some point if const changes aren't being picked up.